internal/monitor/node.go

```diff
@@ -291,6 +298,7 @@ func (pm *PodMonitorType) nodeModeCleanupPods(node *v1.Node) bool {
 
 		// Check containers to make sure they're not running. This uses the containerInfos map obtained above.
 		pod := podInfo.Pod
+		namespace, name := splitPodKey(podKey)
```

Comment:
I'm not sure why you moved this from below the for loop; looks like it doesn't make any difference. Doesn't really matter.

Reply:
I moved that when I thought I needed to check in this function also if pod is driver node pod, forgot to move it down back to original place.

Reply:
reverted